packages/filebrowser-extension/src/index.ts

```diff
@@ -457,3 +455,17 @@ function createContextMenu(path: string, commands: CommandRegistry, registry: Do
 
   return menu;
 }
+
+
+/**
+ * Create a launcher for a given filebrowser widget.
+ */
+function createLauncher(commands: CommandRegistry, widget: FileBrowser): Promise<void> {
+  return commands.execute('launcher:create', {
+    cwd: widget.model.path
+  }).then((launcher: Launcher) => {
+    widget.model.pathChanged.connect(() => {
+      launcher.cwd = widget.model.path;
+    }, launcher);
```

Comment:
Is `launcher` being passed as the second argument here for lifetime management?

Reply:
Yes